android/KMAPro/build.gradle

```diff
@@ -13,14 +17,15 @@ buildscript {
 
         // From jcenter() which could be sunset in future
         // https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/
-        classpath 'com.stepstone.stepper:material-stepper:4.3.1'
+        classpath 'com.stepstone.stepper:material-stepper:4.3.1@aar'
```

Comment:
Line 14 (old) / 18 (new):

> which could be

... so... "has been", now?  Looks like the comment needs an update of some sort.